bodhi/server/consumers/composer.py

```diff
@@ -51,6 +54,9 @@
                                transactional_session_maker)
 
 
+log = logging.getLogger('bodhi')
```

Comment:
This will make the logger show which module emitted the message, which might be a helpful hint:

```suggestion
log = logging.getLogger(__name__)
```